res/values-en-rGB/strings.xml

```diff
@@ -76,6 +76,7 @@
     <string name="auto_silence_never" msgid="4302257878142130807">"Off"</string>
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit